sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -32,71 +32,6 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
   import org.apache.spark.sql.connector.catalog.CatalogV2Util._
 
   override def apply(plan: LogicalPlan): LogicalPlan = plan resolveOperators {
-    case AlterTableAddColumnsStatement(
```

Comment:
This is now moved to `ResolveSessionCatalogs.scala`, thus removing duplicate code. Once all commands are migrated, we should revisit `ResolveSessionCatalogs` and `ResolveCatalogs`.